Plugins/Public/playercntl_plugin/Main.cpp

```diff
@@ -620,10 +646,12 @@ namespace HkIServerImpl
 
 	void __stdcall SystemSwitchOut(uint iClientID, FLPACKET_SYSTEM_SWITCH_OUT& switchOutPacket)
 	{
-		// in case of SERVER_PACKET hooks, first argument is junk data before it gets processed by the server.
-		uint packetClient = HkGetClientIDByShip(switchOutPacket.shipId);
-		if(packetClient)
-			AntiJumpDisconnect::SystemSwitchOut(packetClient);
+		if (iClientID != HkGetClientIDByShip(switchOutPacket.shipId))
```

Comment:
Backwards, I think this comparison is

Reply:
I think it's neater when the variable and name of the function are next to each, as opposed to
`if (HkGetClientIDByShip(switchOutPacket.shipId) != iClientID)`
where iClientID at a glance is compared with switchOutPacker.shipID